nominatim/clicmd/setup.py

```diff
@@ -157,6 +173,11 @@ def run(self, args: NominatimArgs) -> int: # pylint: disable=too-many-statements
 
         return 0
 
+    def _is_complete_import(self, args: NominatimArgs) -> bool:
```

Comment:
You don't really need this extra function. Everywhere where it is used, a simple check for either `continue` or `prepare_database` will do.